backend/spec/mailers/admin_mailer_spec.rb

```diff
@@ -20,4 +22,34 @@
       expect(mail.body.encoded).to match(I18n.t("admin_mailer.farewell_html"))
     end
   end
+
+  describe ".project_developer_created" do
+    let(:mail) { AdminMailer.project_developer_created admin, project_developer }
+
+    it "renders the headers" do
+      expect(mail.subject).to eq(I18n.t("admin_mailer.project_developer_created.subject"))
+      expect(mail.to).to eq([admin.email])
+    end
+
+    it "renders the body" do
+      expect(mail.body.encoded).to match(I18n.t("admin_mailer.greetings", full_name: admin.full_name))
+      expect(mail.body.encoded).to match(I18n.t("admin_mailer.project_developer_created.content_html", new_pd_url: ""))
```

Comment:
I would recommend to test also whole url inside mailer and not just `new_pd_url: ""`. You should be able to use here `link_to('New Investor', backoffice_investor_url(investor))` -- only part which is missing to make this work is I think url helper (`backoffice_investor_url`) which is added automatically to request tests here: https://github.com/Vizzuality/heco-invest/blob/develop/backend/spec/rails_helper.rb#L41. You can modify this line and add these helpers also to mailer tests which should give you all methods you need.